packer/ansible/upgrade.yml

```diff
@@ -1,12 +1,9 @@
 ---
 - hosts: all
-  name: >
-    Set up apt over HTTPS, upgrade base image, and enable automated security
-    updates
+  name: Set up backports, apt over HTTPS, and upgrade base image
   become: yes
   become_method: sudo
   roles:
     - backports
     - apt_over_https
     - upgrade
-    - automated_security_updates
```

Comment:
Should this Ansible role be removed from `requirements.yml`?

Reply:
That role was relocated in https://github.com/cisagov/cyhy_amis/pull/648/commits/45a643848b8c4cd3adea69034e0745ea8e4a83c4 (after being removed in https://github.com/cisagov/cyhy_amis/pull/648/commits/1aa181330f33949b0c9b6ca1084950e2c49f2df9) to mirror [where it is installed in cisagov/skeleton-packer](https://github.com/cisagov/skeleton-packer/blob/develop/src/base.yml). Apologies for that taking place across two commits which made it unclear what was happening.